src/subscriptions/db-service.ts

```diff
@@ -0,0 +1,113 @@
+import { DatabaseTransactionConnection as TrxHandler, sql } from 'slonik';
+
+import { Subscription } from './interfaces/subscription';
+
+/**
+ * Database's first layer of abstraction for Invitations
+ */
+export class SubscriptionService {
+  // the 'safe' way to dynamically generate the columns names:
+  private static allColumns = sql.join(
+    [
+      'id',
+      ['creator', 'creator'],
```

Comment:
is creator the member id ? I think it would be clearer if it's called `memberId`